frontend/packages/pipelines-plugin/src/components/shared/workspaces/WorkspaceDefinitionList.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { PipelineWorkspace } from 'packages/pipelines-plugin/src/types';
+
+export interface WorkspaceDefinitionListProps {
+  workspaces: PipelineWorkspace[];
+}
+
+const WorkspaceDefinitionList: React.FC<WorkspaceDefinitionListProps> = ({ workspaces }) => {
```

Comment:
Do we need this component? Is this not just the same component as the `WorkspaceResourceLinkList` with less bells and whistles? I _think_ they both will render a div list 🤔

Reply:
Yes, the two components need different set of props to function. They also render lists based out of different types of objects. 
There is also a need of certain additional mandatory props in ResourceLinkList which will not be required in DefinitionList